src/components/LineChart/index.tsx

```diff
@@ -30,6 +36,7 @@ export const LineChart: React.FunctionComponent<LineChartProps> = ({
   axisDomainLineColorToken = 'white900',
   visibleAxis = ['left', 'bottom'],
   axisTickPadding = 8,
+  yAxisTopOffsetPercentage,
```

Comment:
it is highly likely that we will need to have some default value for it, if not passed by integrator. 

```suggestion
  yAxisTopOffsetPercentage = 15,
```

Reply:
For this one, I would like to debate. This would be a breaking change, as it changes the existing appearance. Let's leave it without default value, and skip calculations if not defined.